pkgs/development/python-modules/runstats/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  setuptools,
+  cython,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  pytest-cov-stub,
+  pytest-xdist,
+}:
+
+buildPythonPackage rec {
+  pname = "runstats";
+  version = "2.0.0";
```

Comment:
It's recommended that this is set (see [docs](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters)):

```suggestion
  version = "2.0.0";
  pyproject = true;
```

Reply:
Will do. I read that pharagraph as "setting the pyproject=true is recommended for all pyproject python packages.". I think a rewording could help slow people like me :)